github/ci/prow-deploy/kustom/base/manifests/local/github-jira-proxy-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+  name: github-jira-proxy
+  namespace: default
+spec:
+  tls:
+    - hosts:
+        - prow.ci.kubevirt.io
+      secretName: github-jira-proxy-tls
+  rules:
+    - host: prow.ci.kubevirt.io
```

Comment:
We will need to use a different host name here as `prow.ci.kubevirt.io` is already in use by prow.

Reply:
fixed in both places